examples/robot-shop-app/README.md

```diff
@@ -0,0 +1,71 @@
+# Introduction
+
+This is an example online shopping app utilizing microservices in K8s with minor modifications from https://github.com/instana/robot-shop.  The modifications were to set registry helm varaibles and storage.  Note this image by default uses OpenEBS from gravity 7.0+.  On deployment you will get this front page.
+
+![Robot Shop](robotshop.png)
+
+
+## Building Image
+tele build -o robotshop.tar robot-shop-app/resources/app.yaml
+
+## Building App
```

Comment:
Building Application Image. Also, should we briefly explain what "building cluster image" versus "building app image" does?